trackingC.Rmd

```diff
@@ -330,3 +350,35 @@ ggplot(relimp_out, aes(year, value, fill = param)) +
   labs(x = "Year",
        y = "Fraction of relative importance")
 ```
+# Comparing Hector temperature data to CMIP6
+``` {r cmip}
+# Read in comparison data
+cmip <- read.csv("CMIP6_annual_tas_global.csv") %>%
+  # filter by experiment, stop x axis at 2300
+  filter(variable == "Tgav",
+         experiment == "ssp245") %>%
+  group_by(year) %>%
+  summarize(min = min(value),
+         max = max(value),
+         med = median(value)) %>%
+  mutate(source = "CMIP6")
+
+# Extract Hector data
+hector <- results_slice %>%
+  filter(variable == "Tgav",
+         year >= 2015 & year <= 2100) %>%
+  group_by(year, run_number) %>%
+  summarize(min = min(value),
+            max = max(value),
+            med = median(value)) %>%
+  mutate(source = "Hector")
+
+# Combine data
+temp <- bind_rows(cmip, hector)
+
+# Plot temperature spread
+ggplot(temp, aes(year, fill = source)) +
+  geom_ribbon(aes(ymin = min, ymax = max, alpha = 0.5)) +
```

Comment:
Put `alpha = 0.5` _outside_ of the `aes` call. It's a fixed value, not an aesthetic mapped to data.